playbooks/02 - Use Case - OTbase Inventory/OTbase Inventory > Create CVEs.json

```diff
@@ -0,0 +1,228 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "OTbase Inventory > Create CVEs",
+    "aliasName": null,
+    "tag": null,
+    "description": "Creates CVE records and links with the OTbase Inventory Asset",
+    "isActive": true,
+    "debug": false,
+    "singleRecordExecution": false,
+    "remoteExecutableFlag": false,
+    "parameters": [
+        "parentData"
+    ],
+    "synchronous": false,
+    "collection": "\/api\/3\/workflow_collections\/8dbc991d-05bb-45cd-9221-1271f9c06642",
+    "versions": [],
+    "triggerStep": "\/api\/3\/workflow_steps\/bdf8f2e9-db22-44b2-8f60-f47cbf5844ae",
+    "steps": [
+        {
+            "@type": "WorkflowStep",
+            "name": "Configuration",
+            "description": null,
+            "arguments": {
+                "parentData": "{{vars.input.records[0]}}",
+                "sourceData": "{{vars.input.records[0].sourceData}}",
+                "severity_map": "{'CRITICAL': {{'Severity'| picklist('Critical') }},'HIGH': {{'Severity'| picklist('High')}},'MEDIUM': {{'Severity'| picklist('Medium')}},'LOW': {{'Severity'| picklist('Low') }}}"
+            },
+            "status": null,
+            "top": "165",
+            "left": "300",
+            "stepType": "\/api\/3\/workflow_step_types\/04d0cf46-b6a8-42c4-8683-60a7eaa69e8f",
+            "group": null,
+            "uuid": "ed8e4fe8-27bf-4ed0-be9a-3667dae3445a"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Create CVEs",
```

Comment:
@cs-adarsh-gupta , As we discussed, use the "Insert Bulk Feed" step to create CVEs, and then add a separate step to link all CVE records to the asset at once.